util/embedded_target.py

```diff
@@ -68,39 +84,41 @@ def main():
         '-b',
         required=True,
         help='File basename used as preffix in output filenames.')
-    parser.add_argument(
-        '--input',
-        '-i',
-        required=True,
-        help="Input file")
-    parser.add_argument(
-        '--objcopy',
-        '-c',
-        required=True,
-        help='Absolute path to objcopy tool.')
-    parser.add_argument(
-        '--objdump',
-        '-p',
-        required=True,
-        help='Absolute path to objdump tool.')
-    parser.add_argument(
-        '--outdir',
-        '-d',
-        required=True,
-        help="Output directory.")
-    parser.add_argument(
-        '--srec_cat',
-        '-s',
-        required=True,
-        help='Absolute path to srec_cat tool.')
+    parser.add_argument('--input', '-i', required=True, help="Input file")
+    parser.add_argument('--objcopy',
+                        '-c',
+                        required=True,
+                        help='Absolute path to objcopy tool.')
+    parser.add_argument('--objdump',
+                        '-p',
+                        required=True,
+                        help='Absolute path to objdump tool.')
+    parser.add_argument('--outdir',
+                        '-d',
+                        required=True,
+                        help="Output directory.")
+    parser.add_argument('--srec_cat',
+                        '-s',
+                        required=True,
+                        help='Absolute path to srec_cat tool.')
```

Comment:
Yapf doesn't give stable output for these argument lists, please avoid adding them to the diff if not needed.